### ISSUE TITLE
how to solve statuscode:413  Request Entity Too Large 

### ISSUE THREAD

**Issue by Koro-s-ensei** [strapi/strapi#8146]
I have strapi running locally and try to upload video using baseencode 64, but I got the error of 413 whenever my file size is larger than the 1mb. I have created the request.json file inside the ./config/request.json and as well as middleware mentioned in the documentation, but it didn't help me.
config directory
![image](https://user-images.githubusercontent.com/56131320/94889631-8226fd00-049a-11eb-803c-957cb32cb2f1.png)
request.json code mentioned below
`
{
	"parser": {
		"enabled": true,
		"multipart": true,
		"formLimit": 5242880000,
		"jsonLimit": 5242880000,
		"formidable": {
			"maxFileSize": 524288000
		}
	}
}
`
middleware.json
`module.exports = {
  load: {
   after: ['parser'],
  },
};`

**Comment by secretlifeof** [comment]
@Koro-s-ensei can you please edit your answer to use the template. It it easier to read if you use markdown, as an example marking your code. Do you have any errors in your console? Where do you try to upload your file to (aws, etc.)?

**Comment by Koro-s-ensei** [comment]
ok

**Comment by Koro-s-ensei** [comment]
I used post-man  to hit the api to upload the video 
![image](https://user-images.githubusercontent.com/56131320/94909633-1906b000-04c1-11eb-908d-86fdfc45b747.png)
this address of  my repo that i'm working on https://github.com/Koro-s-ensei/demi-2/

**Comment by derrickmehaffy** [comment]
> This is a templated message

Hello, please follow the [issue template](https://raw.githubusercontent.com/strapi/strapi/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md).

A proper issue submission let's us better understand the origin of your bug and therefore help you.

I will reopen your issue when we receive the issue following the template guidelines and properly fill out the template. You can see the template guidelines for bug reports [here](https://github.com/strapi/strapi/blob/master/CONTRIBUTING.md#reporting-an-issue).

Please update the issue with the template and we can reopen this report.

Thank you.

**Comment by Koro-s-ensei** [comment]
https://github.com/strapi/strapi/issues/8157
I have create the new issue according to the issue template.

**Comment by laurenskling** [comment]
I had the same thing, where the docs telling me to add `formLimit` and `jsonLimit` wasn't helping my raw body POST. I dug into the https://github.com/dlau/koa-body#options and found there's also `textLimit` which was the one i needed 👏 

I also had to add includeUnparsed to make sure my XML was getting onto the body.

Hope it helps!

**Comment by ace3** [comment]
note to self, sometimes the issue is on the reverse proxy like Nginx.

**Comment by jacquesh82** [comment]
2 years later, on version 3.6.8, I've the same error. I add a request.json in config dir with this content  ```
{
        "parser": {
                "enabled": true,
                "multipart": true,
                "textLimit":5242880000000,
                "formLimit": 5242880000000,
                "jsonLimit": 5242880000000,
                "formidable": {
                        "maxFileSize": 524288000000
                }
        }
}
```

but no change.

any idea ?

thank you.
